git-remote-hg

```diff
@@ -265,10 +266,10 @@ class Parser:
         return (user, int(date), -tz)
 
 def fix_file_path(path):
-    path = os.path.normpath(path)
+    path = posixpath.normpath(path)
     if not os.path.isabs(path):
```

Comment:
Weird, why is this instance of `os.path` not changed to `posixpath`? Mixing the two like this seems wrong.